src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -190,24 +196,29 @@ public void feederMain() {
         } else if (oi.reverseIntake()) {
             startFeeder(-feederSpeed);
         } else {
+            gathererState = gathererUp;
```

Comment:
This will raise the gatherer all the time, unless you are actively shooting, is that really what you want?

Reply:
fixed